centrallix/tests/test_moneytype_02.c

```diff
@@ -0,0 +1,31 @@
+#include "obj.h"
+#include <assert.h>
+
+long long
+test(char** name)
+{
+    *name = "moneytype_02 - objDataToDouble";
+    double testValue;
```

Comment:
as above, I think it would be more clear if this was something like `testDouble`, or removed entirely